inf-clojure.el

```diff
@@ -702,6 +746,35 @@ is present it will be used instead of the current file."
     (when switch-to-repl
       (inf-clojure-switch-to-repl t))))
 
+(defcustom inf-clojure-reload-default t
```

Comment:
Why not `'reload` and `'reload-all`? Boolean predicate for this seems pretty confusing to me. It's better to use a symbol.

Reply:
I will probably drop this